test.js

```diff
@@ -38,14 +38,12 @@ async function EthStorageTest() {
     console.log(p)
 
     // put blobs
-    // const es = await EthStorage.create({
-    //     rpc: 'http://88.99.30.186:8545',
-    //     privateKey
-    // })
-    // const status = await es.putBlobs(3, content);
-    // console.log(status);
+    const keys = ["key1", "key2"];
+    const blobData = [Buffer.from("some data1"), Buffer.from("some data2")];
+    status = await es.putBlobs(keys, blobData);
+    console.log(status);
 }
-// EthStorageTest();
+EthStorageTest();
```

Comment:
What is the test result? May need to update it in the comment of the PR

Reply:
Additionally, I think the documentation update PRs for web3url and EthStorage need to be prepared and set as dependencies for this PR. The SDK implementation and the documentation have been inconsistent for a long time.

Reply:
Did not find the PR for web3url doc

Reply:
Only find the issue

Reply:
https://github.com/ethstorage/web3url-doc/pull/13